src/electron/ipc/stream.ts

```
@@ -0,0 +1,96 @@
+import { ipcMain } from "electron";
+// import { aiService } from "../services/gemini.js";
+import chalk from "chalk";
+import { ToolHandler } from "../tools/toolHandler.js";
+import { GoogleGenAI } from "@google/genai";
+
+const toolHandler = ToolHandler.getInstance();
+
+export function setupStreamHandlers() {
+  ipcMain.handle("stream-message-with-history", async (event, messages: any[]) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)